prog/vm/nexus.rb

```diff
@@ -293,6 +297,18 @@ def before_run
     hop_wait_sshable
   end
 
+  def self.setup_vm_options_str(frame)
+    options = []
+    options << "hugepages=off" if frame["hugepages"] == false
+    options << "ch_version=#{frame["ch_version"]}" if frame["ch_version"]
+    options << "firmware_version=#{frame["firmware_version"]}" if frame["firmware_version"]
+    options.join(",")
+  end
+
+  def setup_vm_str(action)
```

Comment:
I think it's a bit awkward to expect callers to run this special formatting method and then putting it into sshable.cmd, given it always targets the same sshable, perhaps the ssh access as well should be coupled as a helper (or a few helpers that may share an internal formatting routine) that runs sshable.cmd, e.g. `VmHost[ubid].setup_vm(...)` => dials the VM and does SSH stuff on its own.

Reply:
We could just drop this commit.  Now that it has no special behavior and isn't used by UpdateIpv6, there's not really an advantage for having a separate method.